Changelog.md

```diff
@@ -7,7 +7,7 @@ Compiler Features:
 * JSON-AST: Added selector field for errors and events.
 
 Bugfixes:
-
+ * The compiler uses "C" instead of environment locale.
```

Comment:
This describes how we fixed it but I think that in the changelog we should focus on the effects of the bug visible to the users (i.e. it caused an ICE under certain conditions):
```suggestion
 * Fix internal error for locales with unusual capitalization rules. Locale set in the environment is now completely ignored.
```